<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBElevationDialogBase</class>
 <widget class="QDialog" name="GBElevationDialogBase">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>285</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>583</width>
    <height>285</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>583</width>
    <height>285</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculate Elevations</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QDialogButtonBox" name="button_box">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Digital Terrain Model data folder:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dtmFolder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="dtmFolderButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>. . .</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>DTM grid spacing</string>
   </property>
   <widget class="QRadioButton" name="os10mRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Land-Form PROFILE 10m</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="os50mRadioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Land-Form PANORAMA 50m</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>341</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DTM Files Required:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pointLayersComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Point Layer:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Elevation Attribute:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="attributeComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="attributeButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Interpolation Method:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="interpolationMethodComboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Bilinear Interpolation</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inverse Distance Weighted</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="dtmList">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>301</width>
     <height>211</height>
    </rect>
   </property>
   <property name="alternatingRowColors">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="flow">
    <enum>QListView::LeftToRight</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>true</bool>
   </property>
   <property name="uniformItemSizes">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>GBElevationDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>GBElevationDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
